packages/utils/src/lib/git.integration.test.ts

```diff
@@ -31,7 +31,7 @@ describe('git utils in a git repo', () => {
   describe('without a branch and commits', () => {
     it('getCurrentBranchOrTag should throw if no branch or tag is given', async () => {
       await expect(getCurrentBranchOrTag(emptyGit)).rejects.toThrow(
-        'Could not get current tag or branch.',
+        'No names found, cannot describe anything',
```

Comment:
This message is confusing, as a user I wouldn't know what is going on.

Reply:
When I think about it... I guess I will forward the git error directly.

Reply:
I checked and actually did it already. So what we could do is adding a statement where in the code it happened, but then the stack track gets obsulete.

I suggest we keep the git error message.